src/main/java/backend/resource/TurboMilestone.java

```diff
@@ -83,6 +86,60 @@ public TurboMilestone(TurboMilestone milestone) {
     private void ______METHODS______() {
     }
 
+    public boolean isOverdue() {
+        return dueDate.isPresent() && dueDate.get().isBefore(LocalDate.now());
+    }
+
+    public boolean hasOpenIssues() {
+        return openIssues > 0;
+    }
+
+    /**
+     * A milestone is ongoing if it is open and not due yet or if it is overdue but still open and has open issues.
+     */
+    public boolean isOngoing() {
+        return isOpen() && (!isOverdue() || hasOpenIssues());
+    }
+
+    public static List<TurboMilestone> filterMilestonesOfGivenRepoIds(List<TurboMilestone> milestones,
```

Comment:
@HansNewbie suggested to change it to `filterMilestonesOfGivenRepoIds` since it's more like "filtering" rather than "getting". Should I change it back?